src/components/Notification/subcomponents/NotificationsContainer/NotificationsContainer.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+import { notificationsContainer } from './NotificationsContainer.style';
+
+type Positions = 'top-right' | 'top-left' | 'bottom-left' | 'bottom-right';
+
+type Props = {
+  /** Notifications Container children */
+  children: React.ReactNode[];
+  /** Notifications Container position */
+  position: Positions;
+};
+
+const positionToBooleans = (position: Positions) =>
```

Comment:
To avoid the complexity of the nested conditions you can export the position logic to an object.

Example: 
```
const positionOptions = {
 'top-right': [true, false, false, true],
'top-left': [true, false, false, true],
'bottom-left': [false, true, true, false],
'bottom-right': [false, true, false, true]
};

//in the Container you will have...
const currentPosition = positionOptions[position];
return <div css={notificationsContainer(currentPosition)}>{children}</div>;
```

or even better using a factory function
```
const getPositionStyle = (styleLiteral: string) => {
  return css`
    ${styleLiteral}
  `;
};
const positionOptions = {
  'top-right': `top:0; right: 0;`,
  'top-left': `top:0; left: 0;`,
  'bottom-left': `bottom:0; left: 0;`,
  'bottom-right': `bottom:0; right: 0;`,
};

use it like this...
export const notificationsContainer = (currentPositon: Positions) => (
  theme: Theme
): SerializedStyles => css`
   position: absolute;
 .... 
  ${getPositionStyle(positionOptions[currentPositon])}
`;
```